src/app/services/central-server.service.ts

```diff
@@ -2366,7 +2366,7 @@ export class CentralServerService {
     // Set the tenant
     params['Tenant'] = this.windowService.getSubdomain();
     // Execute the REST service
-    return this.httpClient.get<ActionResponse>(`${this.centralRestServerServiceAuthURL}/${ServerAction.VERIFY_EMAIL}`,
+    return this.httpClient.get<ActionResponse>(`${this.restServerAuthURL}/${ServerAction.VERIFY_EMAIL}`,
```

Comment:
That endpoint does not exist on the new REST API. 

And the ServerAction does not follow the naming scheme defined in the issue relative to the new API definition. 

Have you tested your changes? because I doubt it works.